pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -103,50 +106,48 @@ func (m topologyToMatchedTermCount) clone() topologyToMatchedTermCount {
 	return copy
 }
 
+func (m topologyToMatchedTermCount) updateWithTerm(node *v1.Node, t *framework.AffinityTerm, value int64) {
```

Comment:
nit: only affinityTerm.TopologyKey is needed 

```suggestion
func (m topologyToMatchedTermCount) updateWithTerm(node *v1.Node, tk string, value int64) {
```